### PR TITLE
Update dev guide for benchmarks

### PR DESCRIPTION
**Pull request by MiguelWeezardo**
## Description

> Is this change a fix, improvement, new feature, refactoring, or other?

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

> How would you describe this change to a non-technical end user or system administrator?

## Documentation

( ) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py [hunk 1]
```diff
@@ -1,53 +1,84 @@
...
+        ('tpcds_sf10_orc', 'tpcds.sf10'),
+        ('tpcds_sf30_orc', 'tpcds.sf30'),
+        ('tpcds_sf100_orc', 'tpcds.sf100'),
+        ('tpcds_sf300_orc', 'tpcds.sf300'),
+        ('tpcds_sf1000_orc', 'tpcds.sf1000'),
+        ('tpcds_sf3000_orc', 'tpcds.sf3000'),
+        ('tpcds_sf10000_orc', 'tpcds.sf10000'),
+    ]
+
+    tables = [
```
Comment (nineinchnick) [inline testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py:17]: This overrides the `tables` argument and `formats` looks unused.

File: testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py [hunk 2]
```diff
@@ -1,53 +1,84 @@
...
+        for table in tables:
+            print('CREATE TABLE IF NOT EXISTS "hive"."{}"."{}" WITH (format = \'{}\') AS SELECT * FROM {}."{}";'.format(
+                  new_schema, table, format, source_schema, table))
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Generate test data.')
+    parser.add_argument('--factors', type=csvtype(['tiny', 'sf1', 'sf10', 'sf30', 'sf100', 'sf300', 'sf1000', 'sf3000']), default=['sf10', 'sf30', 'sf100', 'sf300', 'sf1000', 'sf3000'])
+    parser.add_argument('--formats', type=csvtype(['orc', 'text']), default=['orc'])
+    default_tables = ['customer', 'lineitem', 'nation', 'orders', 'part', 'partsupp', 'region', 'supplier']
```
Comment (nineinchnick) [inline testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py:63]: These tables are for `tpch`, not `tpcds`

File: testing/trino-benchto-benchmarks/README.md [hunk 3]
```diff
@@ -18,7 +18,8 @@ features. Therefore benchmarks are organized in suites, like:
 
 ### Requirements
 
-* Presto already installed on the target environment
+* JDK 8 for compiling and running Benchto
+* Trino already installed on the target environment
```
Comment (findepi) [inline testing/trino-benchto-benchmarks/README.md:22]: This shouldn't be part of `Pin java version for trino-benchto-benchmarks module to JDK 8` commit

File: testing/trino-benchto-benchmarks/README.md [hunk 4]
```diff
@@ -78,17 +82,21 @@ runs or different underlying schemas. Create a simple `overrides.yaml` file:
 
 ```yaml
 runs: 10
-tpch_medium: tpcds_10gb_txt
+tpch_300: tpch_sf1_orc
+scale_300: 1
+tpch_1000: tpch_sf1_orc
+scale_1000: 1
+tpch_3000: tpch_sf1_orc
```
Comment (findepi) [inline testing/trino-benchto-benchmarks/README.md:89]: Let's do trivial renames (Presto -> Trino) separarely from other changes

File: testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py [hunk 5]
```diff
@@ -1,53 +1,63 @@
 #!/usr/bin/env python
```
Comment (sopel39) [inline testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py:1]: did you use the tool to generate schemas manually as a test?

**Review by sopel39 — COMMENTED** [review]
lgtm % @findepi comments

File: testing/trino-benchto-benchmarks/pom.xml [hunk 6]
```diff
@@ -12,6 +12,9 @@
     <name>trino-benchto-benchmarks</name>
 
     <properties>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```
[the remark this answers is not in the document]
Comment (MiguelWeezardo) [reply testing/trino-benchto-benchmarks/pom.xml]: It's not, removed

File: testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py (hunk 5 above)
Comment (MiguelWeezardo) [reply testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py:1]: Thanks, running this unveiled a trivial bug.

**Review by nineinchnick — APPROVED** [review]

**Comment by sopel39** [comment]
Failed due: https://github.com/trinodb/trino/issues/14618

**Pull request merged**